pylintrc

```diff
@@ -89,7 +89,9 @@ disable=abstract-method,
         long-suffix,
         map-builtin-not-iterating,
         misplaced-comparison-constant,
+        missing-class-docstring,
```

Comment:
class and module doc string should be available

Reply:
Done.